test/e2e/common_test.go

```diff
@@ -234,9 +234,18 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 	if os.Getenv("CONMON_BINARY") != "" {
 		conmonBinary = os.Getenv("CONMON_BINARY")
 	}
+
+	storageFs := STORAGE_FS
+	if rootless.IsRootless() {
+		storageFs = ROOTLESS_STORAGE_FS
+	}
+
 	storageOptions := STORAGE_OPTIONS
 	if os.Getenv("STORAGE_OPTIONS") != "" {
 		storageOptions = os.Getenv("STORAGE_OPTIONS")
+		if !strings.Contains(storageOptions, storageFs) {
+			storageFs = strings.Split(storageOptions, " ")[1]
```

Comment:
This, in particular, is unwise: you are assuming that `STORAGE_OPTIONS` will only be `--storage-driver <space> <driver>`. This will fail catastrophically if `STORAGE_OPTIONS` is `--abc`, or `--this=that`, or `--another-option x`.

I see no actual use of `STORAGE_OPTIONS` anywhere in the source tree. What I would recommend is getting rid of the option entirely, and relying solely on `STORAGE_FS`. If `STORAGE_FS` is set, then set `storageOptions` to `--storage-driver $STORAGE_FS`.